docs/Deploying/Load-testing.md

```diff
@@ -14,33 +14,35 @@ A test is deemed successful when the Fleet server is able to receive and make re
 
 ## Results
 
-### 1,000 hosts
+### 2,500 hosts
 
 With the following infrastructure, 1,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instances| CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 1 Fargate task | 256 CPU Units  |512 MB of memory|
+| 1 Fargate task | 512 CPU Units  | 4GB of memory |
 
 |&#8203;| Version                 |Instance type |
 |-------|-------------------------|--------------|
-| Redis | 5.0.6                   |cache.m5.large|
-| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.medium|
+| Redis | 5.0.6                   | cache.t4g.medium |
+| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.small |
 
 ### 150,000 hosts
 
 With the infrastructure listed below, 150,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instance | CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 25 Fargate tasks | 1024 CPU units  |2048 MB of memory|
+| 20 Fargate tasks | 1024 CPU units  | 4GB of memory |
 
-|&#8203;| Version                 |Instance type |
-|-------|-------------------------|--------------|
-| Redis | 5.0.6                   |cache.m5.large|
-| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.medium|
+|&#8203;| Version                 |Instance type   |
+|-------|-------------------------|----------------|
+| Redis | 5.0.6                   | cache.m6g.large |
+| MySQL | 5.7.mysql_aurora.2.10.0 | db.r6g.8xlarge (with RR) |
```

Comment:
This is a steep change, we did add a bunch of features, but we also added a ton of perf improvements. Would love to understand more what pushed for this instance (plus a RR).

Reply:
Distributed read was a huge user along with host last seen at. I have no idea how it ran at 150K under this architecture before given my findings.

Reply:
I wonder if we added something to make this the case... everything has pointed at needing less to run the same, not more. This warrants further investigation, because it feels like a big regression that we missed.

Reply:
Ok, we can call it that, but certainly we didn't need a RR at that point.

Reply:
Ah, so the system that removes old hosts didn't run which caused additional load. Rerunning a loadtest at 150K with only those hosts on it only required a 4xlarge.